how-to/how-to-template.md

```diff
@@ -0,0 +1,54 @@
+<!-- Copy this Template. -->
+<!-- Describe the title of your article by replacing "How To Template" with the page name you want to publish to. -->
+
+# Title
+
+::information_source: Before using this template, read the accompanying [how-to template guide](how-to-template-guide.md)
```

Comment:
**nitpick:** A colon (:) appears at the beginning of the sentence in the output.

Reply:
Fixed